.ci/containers/gcb-terraform-vcr-tester/Dockerfile

```diff
@@ -0,0 +1,11 @@
+from golang:1.16-stretch as resource
```

Comment:
I think we want to use the same (or similar) image as the downstream builder or the terraform unit tester/linter. A fresh golang image won't have Terraform core installed